packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,209 @@
+} satisfies Meta<typeof Textarea>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: `Styling via the \`.utrecht-textarea\` class name.`,
+      },
+    },
+  },
+};
+
+export const DefaultValue: Story = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => {
+    return <Textarea defaultValue={args.children} />;
+  },
+};
+
+export const PlaceholderRightToLeft: Story = {
+  name: 'Placeholder (right-to-left)',
+  args: {
+    ...Default.args,
+    placeholder: 'اكتب رسالتك',
+    children: '',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const RightToLeft: Story = {
+  args: {
+    ...Default.args,
+    children:
+      'لكن لا بد أن أوضح لك أن كل هذه الأفكار المغلوطة حول استنكار  النشوة وتمجيد الألم نشأت بالفعل، وسأعرض لك التفاصيل لتكتشف حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانية ومنطقية فيعرضهم هذا لمواجهة الظروف الأليمة، وأكرر بأنه لا يوجد من يرغب في الحب ونيل المنال ويتلذذ بالآلام، الألم هو الألم ولكن نتيجة لظروف ما قد تكمن السعاده فيما نتحمله من كد وأسي.',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const Spellcheck: Story = {
+  args: {
+    ...Default.args,
+    spellCheck: true,
+  },
+};
+
+export const Disabled: Story = {
+  args: {
+    ...Default.args,
+    disabled: true,
+  },
+};
+
+export const Invalid: Story = {
+  args: {
+    ...Default.args,
+    invalid: true,
+  },
+};
+
+export const ReadOnly: Story = {
+  args: {
+    ...Default.args,
+    readOnly: true,
+  },
+};
+
+export const Required: Story = {
+  args: {
+    ...Default.args,
+    required: true,
+  },
+};
+
+export const Focus: Story = {
+  args: {
+    ...Default.args,
+    focus: true,
+    focusVisible: true,
+  },
+  parameters: {
+    ...Default.parameters,
+    docs: {
+      ...Default.parameters?.['docs'],
+      source: {
+        code: `<textarea class="utrecht-textarea utrecht-textarea--html-textarea utrecht-textarea--focus utrecht-textarea--focus-visible">
+                 The Quick Brown Fox Jumps Over The Lazy Dog
+               </textarea>`,
+      },
+    },
+  },
+  render: (args) => {
+    return (
+      <Textarea
+        {...args}
+        className={`${args.focus && 'utrecht-textarea--focus'} ${
+          args.focusVisible && 'utrecht-textarea--focus-visible'
+        }`}
+      />
+    );
+  },
+};
+
+export const Hover: Story = {
+  args: {
+    ...Default.args,
+    hover: true,
+  },
+  parameters: {
+    ...Default.parameters,
+    docs: {
+      ...Default.parameters?.['docs'],
+      source: {
+        code: `<textarea class="utrecht-textarea utrecht-textarea--html-textarea utrecht-textarea--hover">
+                 The Quick Brown Fox Jumps Over The Lazy Dog
+               </textarea>`,
```

Comment:
This code formatting would be fixed too, using Prettier:

<img width="1016" alt="Screenshot 2023-08-31 at 08 36 09" src="https://github.com/nl-design-system/utrecht/assets/30694/4f26cf67-e0c3-43f0-83e6-dfd212b9f6d2">

If we didn't use Prettier, you could also use a utility such as [ts-dedent](https://www.npmjs.com/package/ts-dedent).

Reply:
for the record, I ran prettier. but I'll fix it up